js/common/README.md

```diff
@@ -0,0 +1,13 @@
+# ONNX Runtime JavaScript API
+
+ONNX Runtime JavaScript API is a unified API for all JavaScript usages. It's dependency of the following NPM packages:
+
+- onnxruntime-node
+- onnxruntime-web
+- onnxruntime-react-native
+
+ONNX Runtime JavaScript API is not designed for using directly. Please consider to install one of the NPM packages above according to target platform.
```

Comment:
Actually, a user will use ONNX Runtime JavaScript API directly. It's not designed for standalone library?

Reply:
This library, onnxruntime-common itself, is a standalone library. it does not have extra dependency for itself to work correctly

"not designed for using directly" here means users usually want to have at least one backend implementation to actually inference an onnx model.

Reply:
updated readme.